### ISSUE TITLE
Waveform UI improvement ideas

### ISSUE THREAD

**Issue by Phyks** [ampache/ampache#1295]
These are some issues collected from our users.

- [ ] When the waveform can't be computed, one see only a "broken image". I guess a better way would be to hide the whole line.

- [ ] Clicking on the waveform could (should?) seek in the music.

**Comment by Afterster** [comment]
I agree for the first but as the waveform is calculated when loading it the first time, I'm not sure we can do it without ugly trick.
For the second, it's already the case if you click on waveform on the player.

**Comment by Niols** [comment]
I think the first one could be easely fixed with javascript loading of the image.
But then I don't know if you consider it to be an ugly trick or not ^^

For the second, it doesn't seem to be working on my instance. Do I need to enable something?

**Comment by Phyks** [comment]
I think this kind of snippet would do the trick to handle missing waveform:
```
<script language='javascript'>
function defaultImage(img)
{
    img.onerror = "";
    img.src = 'default.gif';
}
</script>
<img alt="My Image" src="someimage.gif" onerror="defaultImage(this);" />
```

And eventually replace it by a text stating that waveform could not be computed, or simply delete it.

**Comment by producerism** [comment]
I've noticed this issue too, with clicking on the waveform not having any effect. For me, it only seems to happen with transcoded audio files (e.g. FLAC). When this happens, the regular seekbar (jp-seek-bar) also remains gray and never updates. Clicking on it also has no effect.

**Comment by Phyks** [comment]
@producerism Not sure this is the same, and yours look to be more related to https://github.com/ampache/ampache/issues/1303.
